doc/man/bitcoind.1

```diff
@@ -90,7 +90,7 @@ Reduce storage requirements by enabling pruning (deleting) of old
 blocks. This allows the pruneblockchain RPC to be called to
 delete specific blocks, and enables automatic pruning of old
 blocks if a target size in MiB is provided. This mode is
-incompatible with \fB\-txindex\fR and \fB\-rescan\fR. Warning: Reverting this
+incompatible with \fB\-txindex\fR. Warning: Reverting this
```

Comment:
I think these files are automatically generated from -help output and could be reverted in the PR.